samples/src/cpp-baremetal-semihosting-prof/README.md

```diff
@@ -0,0 +1,15 @@
+# Code profiling and coverage sample
+
+This sample shows how to use instrumentation to emit profile data 
+and use it to show code coverage.
+
+This sample makes use of LLVM tools `llvm-profdata` and `llvm-cov` 
+that are not provided with LLVM Embedded Toolchain for Arm, 
```

Comment:
Looking at https://github.com/ARM-software/LLVM-embedded-toolchain-for-Arm/blob/main/CMakeLists.txt#L140 it looks like Linux and MacOS versions will, but I'm not sure about the mingw ones. Looks like something that we might be able to fix.

Reply:
My bad, it is provided in all variants - let me fix this.